routers/web/org/projects.go

```diff
@@ -436,9 +436,11 @@ func UpdateIssueProject(ctx *context.Context) {
 
 	projectID := ctx.FormInt64("id")
 	for _, issue := range issues {
-		oldProjectID := issue.Project.ID
-		if oldProjectID == projectID {
-			continue
+		if issue.Project != nil {
+			oldProjectID := issue.Project.ID
+			if oldProjectID == projectID {
```

Comment:
What's the point of assigning a new variable just for the if check?
Why not just do
```suggestion
			if issue.Project.ID == projectID {
```

Reply:
I didn't notice this. Just added Line 439 here. 😄

But,
in old code, `oldProjectID` comes from `issue.ProjectID()` which is a function.
Maybe we don't need this any more.